css-four-column-layout/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>css-four-column-layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@300;700&display=swap" rel="stylesheet">
+</head>
+<body class="containter">
+  <div class="row">
+    <h2>How It Works</h2>
+  </div>
+  <div class="row">
+    <div class="column">
+      <img class="phone-image" src="images/screen-1.png">
+    </div>
+    <div class="column">
+      <img class="phone-image" src="images/screen-2.png">
+    </div>
+    <div class="column">
+      <img class="phone-image" src="images/screen-3.png">
+    </div>
+    <div class="column">
+      <img class="phone-image" src="images/screen-4.png">
+    </div>
+  </div>
+  <div class="row">
+    <div class="column">
+      <img class="icon" src="images/switch-icon.png" alt="switch-icon">
+      <h3 class="text-center">App Switching</h3>
+      <p class="text-center">Toggles all your apps on and off.</p>
+    </div>
+    <div class="column">
+      <img class="icon" src="images/filter-icon.png" alt="filter-icon">
+      <h3 class="text-center">Trip Filtering</h3>
+      <p class="text-center">Evaluates your trip requests.</p>
+    </div>
+    <div class="column">
+      <img class="icon" src="images/slider-icon.png" alt="slider-icon">
+      <h3 class="text-center">Service Filters</h3>
+      <p class="text-center">Accepts your ideal trips.</p>
+    </div>
+    <div class="column">
+      <img class="icon" src="images/bars-icon.png" alt="bars-icon">
+      <h3 class="text-center">Stats + Mileage</h3>
+      <p class="text-center">Tracks your trips across all apps.</p>
+    </div>
+  </div>
```

Comment:
I can see why you would do this from the perspective of layout classes, but by putting the images, icons, and text in separate rows, your HTML looks like this content is not related to one another. You should have one row with four columns, and each column contains the phone image, icon, and text all together.